### PR TITLE
Style tweak the crd task file

### PR DESCRIPTION
**Pull request by tengqm**
This PR reformats the CRD task file where line wrappings are expected or proper indentations are needed. It also fixes the syntax code for some code snippets, especially those that look like YAML but are actually not YAML.
Those snippets are some YAML plus some dots. They cannot be used in place of a valid YAML file. We don't need to syntax highlight them as YAML. Just rendering them as plain code would suffice.

### CODE REVIEW

File: content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md [hunk 1]
```diff
@@ -221,7 +221,7 @@ A structural schema is an [OpenAPI v3.0 validation schema](#validation) which:
 
 Non-structural example 1:
 
-```yaml
+```none
```
Comment (sftim) [inline content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md:224]: The commit message doesn't explain this change - at least, I couldn't work out the motivation.

File: content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md [hunk 2]
```diff
@@ -1729,19 +1750,17 @@ kubectl apply -f my-crontab.yaml
...
 
-```
+```shell
 kubectl get all
 ```
 
 and it will include the custom resources of kind `CronTab`:
 
-```console
+```none
```
Comment (sftim) [inline content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md:1759]: Why this change?

File: content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md:224]: These snippets are not YAML, they are just snippets of YAML plus some dots. They cannot be used in place of a valid YAML file. We don't need to syntax highlight them as YAML. Just rendering them as plain code would suffice.

Comment (sftim) [reply content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md:224]: I recommend explaining that in the commit message and PR description (doesn't have to be a long explanation).

Comment (tengqm) [reply content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md:224]: Done.

**Comment by sftim** [comment]
/kind cleanup
/lgtm

Relevant to https://github.com/kubernetes/website/issues/35991
